linux_os/guide/system/auditing/configure_auditd_data_retention/auditd_local_events/rule.yml

```diff
@@ -36,6 +36,9 @@ ocil: |-
     The output should return the following:
     <pre>local_events = yes</pre>
 
+fixtext: |-
+    {{{ fixtext_audit_configuration(param="local_events", value="yes") | indent(4) }}}}
```

Comment:
There are 4 right braces but there should be 3.

This problem repeats.

Reply:
fixed